src/renderers/webgl/WebGLInfo.js

```diff
@@ -13,30 +15,52 @@ function WebGLInfo( gl ) {
 		lines: 0
 	};
 
+	const frameBuffer = {
+		frame: 0,
+		calls: 0,
+		triangles: 0,
+		points: 0,
+		lines: 0
+	};
+
+	function setRenderTarget( isRenderTarget ) {
```

Comment:
I'm not so happy with this method name. `WebGLRenderer.setRenderTarget()` accepts a render target, this one a boolean. That seems not consistent to me.

Reply:
setIsRenderTarget()  --> In that case we rename framebuffer `rtt`

setIsRTT() --> In that case we rename framebuffer `rtt`

setOffScreen() --> In that case we rename framebuffer `offscreen`


What do you think about these options? @Mugen87 @mrdoob

Reply:
I would go for a simple public `rttEnabled` flag which is set in `WebGLRenderer`. No method.

Reply:
I would go for an internal `rtt` flag which is enabled/disabled by `enabledRTT()` and `disableRTT()`. Yes, two methods instead of one but it seems more readable/descriptive to me.